crates/gpui/Cargo.toml

```diff
@@ -68,6 +68,7 @@ usvg = { version = "0.14", features = [] }
 util.workspace = true
 uuid = { version = "1.1.2", features = ["v4"] }
 waker-fn = "1.1.0"
+rfd = "0.13.0"
```

Comment:
If we go with this I think we only want to enable it on linux

Reply:
We'll probably also want to use the `xdg-desktop-portal` backend instead of the default GTK3, since the portal is required for correct behaviour in both Flatpak and Wayland. Seems that requires enabling either the `tokio` or `async-std` feature in the crate, though. Tokio is out since Zed has its own async executor, but I don't know if async-std can be used with that?

Reply:
I think this is probably good enough to start off with. But yeah, we'll have to write our own bindings to xdg-desktop-portal. Or contribute a GPUI-compatible executor interface.

Reply:
Yes `xdg-desktop-portal` is the way to go, it's going to use the native dialog and use the correct user options. It seems to work with `async-std`.

Reply:
I've selected [ahspd](https://github.com/bilelmoussaoui/ashpd/tree/master) instead after consideration, it's the crate that powers `rfd`'s `xdg-desktop-portal` backend, it's a bit lower-level, and also more ergonomic to use. It also covers a much wider range of desktop services that could be useful in the future (e.g. notifications).